designsystem/src/main/res/drawable/ic_validator.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="16dp"
+    android:viewportWidth="16"
+    android:viewportHeight="16">
+  <group>
+    <clip-path
+        android:pathData="M0,0h16v16H0z"/>
+    <path
+        android:pathData="M13.47,4.414v6.695a5.804,5.804 0,0 0,-0.707 -0.365L12.763,4.732c0.258,-0.098 0.495,-0.205 0.706,-0.318zM11.563,5.096v5.233c-1.03,-0.28 -2.251,-0.442 -3.56,-0.442 -1.31,0 -2.533,0.162 -3.563,0.442L4.44,5.096c1.03,0.245 2.252,0.386 3.563,0.386 1.309,0 2.53,-0.141 3.56,-0.386zM12.546,3.526c0.385,-0.164 0.647,-0.33 0.803,-0.468 0.033,-0.03 0.059,-0.055 0.078,-0.076a1.04,1.04 0,0 0,-0.078 -0.075c-0.188,-0.167 -0.53,-0.372 -1.054,-0.569 -1.04,-0.39 -2.557,-0.656 -4.292,-0.656 -1.736,0 -3.253,0.266 -4.293,0.656 -0.525,0.197 -0.866,0.402 -1.054,0.57 -0.033,0.029 -0.058,0.054 -0.077,0.074 0.019,0.021 0.044,0.046 0.077,0.076 0.156,0.138 0.418,0.303 0.802,0.468 0.106,-0.164 0.25,-0.3 0.433,-0.394 0.725,-0.371 2.072,-0.864 4.11,-0.864 2.04,0 3.386,0.493 4.111,0.864 0.183,0.094 0.328,0.23 0.434,0.394zM4.937,3.978c0.65,-0.251 1.66,-0.51 3.064,-0.51 1.406,0 2.415,0.259 3.066,0.51 -0.879,0.19 -1.925,0.304 -3.064,0.304 -1.14,0 -2.187,-0.114 -3.066,-0.304zM3.24,4.732a6.138,6.138 0,0 1,-0.704 -0.318v6.695c0.211,-0.13 0.447,-0.251 0.704,-0.364L3.24,4.732zM3.29,12.057c-0.26,0.146 -0.45,0.29 -0.576,0.418a0.819,0.819 0,0 0,-0.16 0.21,0.145 0.145,0 0,0 -0.018,0.059v0.001c0,0.004 0,0.021 0.018,0.058 0.021,0.044 0.066,0.116 0.16,0.21 0.193,0.197 0.533,0.428 1.047,0.649l0.025,0.01c0.965,0.318 2.359,0.61 4.217,0.61 1.858,0 3.25,-0.292 4.216,-0.61l0.025,-0.01c0.514,-0.22 0.854,-0.452 1.047,-0.648a0.82,0.82 0,0 0,0.16 -0.21c0.018,-0.04 0.018,-0.056 0.018,-0.06 0,-0.003 0,-0.02 -0.018,-0.058a0.82,0.82 0,0 0,-0.16 -0.21,2.606 2.606,0 0,0 -0.578,-0.42 1.19,1.19 0,0 1,-0.6 0.777c-0.726,0.371 -2.073,0.864 -4.112,0.864 -2.038,0 -3.385,-0.493 -4.11,-0.864a1.19,1.19 0,0 1,-0.6 -0.776zM11.563,11.578c-0.961,-0.3 -2.191,-0.49 -3.56,-0.49 -1.37,0 -2.602,0.19 -3.563,0.49v0.08c0,0.06 0.011,0.099 0.02,0.117 0.573,0.29 1.727,0.722 3.541,0.722 1.815,0 2.969,-0.431 3.543,-0.722a0.309,0.309 0,0 0,0.02 -0.117v-0.08zM13.988,14.233a9.26,9.26 0,0 1,-1.347 0.564c-1.2,0.498 -2.836,0.805 -4.638,0.805 -1.803,0 -3.439,-0.307 -4.639,-0.805a9.26,9.26 0,0 1,-1.347 -0.564c-0.449,-0.234 -0.681,-0.712 -0.681,-1.218L1.336,2.95c0,-0.506 0.232,-0.984 0.68,-1.218 0.947,-0.494 2.9,-1.25 5.987,-1.25 3.086,0 5.04,0.756 5.985,1.25 0.45,0.234 0.681,0.712 0.681,1.218v10.065c0,0.506 -0.232,0.984 -0.68,1.218z"
+        android:fillColor="#8A8FA4"
+        android:fillType="evenOdd"/>
+  </group>
+</vector>
```

Comment:
There is the same icon in app, do you mind deleting the app one and using this in the whole project, instead of having two?